shiny/render/_render.py

```diff
@@ -560,3 +564,97 @@ def ui(
     ~shiny.ui.output_ui
     """
     return UiTransformer(_fn)
+
+
+# ======================================================================================
+# RenderDownload
+# ======================================================================================
+
+
+def download_button_wrapped(id: str) -> Tag:
+    return _ui.download_button(id, label=id)
+
+
+@output_transformer(
+    default_ui=download_button_wrapped
+)  # pyright: ignore[reportGeneralTypeIssues]
+async def DownloadTransformer(
+    _meta: TransformerMetadata,
+    _fn: DownloadHandler,
+    *,
+    button_label: Optional[TagChild] = None,
```

Comment:
The `button_label` gets passed here, but it currently isn't used, and doesn't end up in the UI. It would need to somehow go to the call to `download_button_wrapped()`.

Reply:
This should work great in #964